README.md

```diff
@@ -24,14 +24,31 @@ code will create the user with the appropriate name and permissions.  This only
 needs to be run once per project, per AWS account.  This user will also be used by
 GitHub Actions.
 
+Before the build user can be created, the following profiles must exist in
+your AWS credentials file:
+
+* `cool-images-provisionec2amicreateroles`
+* `cool-images-provisionparameterstorereadroles`
+* `cool-terraform-backend`
+* `cool-users-provisionaccount`
+
+The easiest way to set up those profiles is to use our
+[aws-profile-sync](https://github.com/cisagov/aws-profile-sync) utility.
```

Comment:
I think for consistency the two references to the utility in this paragraph should use `code` formatting.

Reply:
I made these changes in `skeleton-packer-cool` repo (see https://github.com/cisagov/skeleton-packer-cool/commit/afc9eaeef402ebae2829c366dbe0eac9dd99e008), so they will get pulled in to this repo at some point in the future.